keps/sig-node/2400-node-swap/README.md

```diff
@@ -731,11 +826,13 @@ Here are specific improvements to be made:
 
 #### GA
 
-_(Tentative.)_
-
-- Test a wide variety of scenarios that may be affected by swap support.
-- Remove feature flag.
-- Remove the Swap Support using Burstable QoS Pods only deprecated in Beta 2.
+- Test a wide variety of scenarios that may be affected by swap support, including tests with aggressive memory stress.
+- Address memory-backed backed volumes which should not have access to swap.  
+- Remove feature gate.
+- Increase debug abilities by introducing a new swap condition to the node.
```

Comment:
SWAP condition is not helpful to signal presence. For presence I wonder if we should rather use a label.
The condition would be good to indicate trouble.

Reply:
This is an ongoing discussion that's taking place in https://github.com/kubernetes/kubernetes/pull/123963 and https://github.com/kubernetes/kubernetes/pull/125278, so I feel this is not the right place to determine these specifics.

I've reworded this requirement to the following in order to capture the main idea instead of outlining the specifics by which it would be implemented:
>Address increasing debug abilities by providing a method to easily understand which nodes are swap-enabled.